### PR TITLE
exec: update usage example

### PR DESCRIPTION
**Pull request by bamarni**
The usage changed after some refactoring (https://github.com/dcos/dcos-go/commit/52d35f5a9660aff52a3cd0fcb933d059f769b5e5).

### CODE REVIEW

**Comment by bamarni** [comment]
@collinvandyck : any chance you could confirm this?

File: exec/exec.go [hunk 1]
```diff
@@ -12,12 +12,14 @@ import (
...
-// ce := exec.Run("bash", []string{"infinite.sh"}, exec.Timeout(3 * time.Second))
+// ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+// defer cancel()
 //
+// ce, _ := exec.Run(ctx, "bash", []string{"infinite.sh"})
 // io.Copy(os.Stdout, ce)
 // err := <- ce.Done
 // if err != nil {
-// 	log.Error(err)
+// 	log.Fatal(err)
```
Comment (collinvandyck) [inline exec/exec.go]: Why did you change this to `log.Fatal`?  Just curious....

File: exec/exec.go [hunk 2]
```diff
@@ -12,12 +12,14 @@ import (
 // dcos-go/exec is a os/exec wrapper. It implements io.Reader and can be used to read both STDOUT and STDERR.
 //
 // Usage:
-// ce := exec.Run("bash", []string{"infinite.sh"}, exec.Timeout(3 * time.Second))
+// ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+// defer cancel()
 //
+// ce, _ := exec.Run(ctx, "bash", []string{"infinite.sh"})
```
Comment (collinvandyck) [inline exec/exec.go]: I think we should catch the error here and handle it to show the possibility that `exec.Run` could not actually invoke the command.

Comment (bamarni) [inline exec/exec.go]: Indeed it feels wrong to ignore the error, I think I had left it because that function implementation never returns any error at the moment. Do you mean this might be here for later implementation?

File: exec/exec.go (hunk 1 above)
Comment (bamarni) [inline exec/exec.go]: I assumed this was the log package from the std library? Which doesn't have `Error()`.

File: exec/exec.go (hunk 2 above)
Comment (collinvandyck) [inline exec/exec.go]: Yeah, returning the error is part of the `exec.Run` API, so we should handle it in the example. I also wondered why the API returned an error but the code did not and I'm guessing that you're right that it was left there for later implementation.

File: exec/exec.go (hunk 1 above)
Comment (collinvandyck) [inline exec/exec.go]: Ah, interesting. Perhaps a `log.Println` makes more sense here?  I could go either way on that -- whatever you think is best is fine.

**Comment by bamarni** [comment]
just updated according to your comments
